src/main/java/lab3/game/Board.java

```diff
@@ -0,0 +1,78 @@
+package lab3.game;
+
+
+import java.util.Arrays;
+
+public class Board {
+    private final char[][] grid;
+    private static final int SIZE = 3;
+    private static final char EMPTY = ' ';
+
+    public Board() {
+        grid = new char[SIZE][SIZE];
+        for (char[] row : grid) {
+            Arrays.fill(row, EMPTY);
+        }
+    }
+
+    public boolean placeMove(int row, int col, char symbol) {
```

Comment:
Again, using a `Position` record as in the sample solution prevents the possibility of calling this method using row/col values that are out of range